test.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 
-set -e -x
+set -e
 
+export PATH=./bin:$PATH
 export RAILS_ENV=test
 export BUNDLE_WITHOUT=development
 
-bundle install --quiet
-bundle exec bowndler install
+bundle install
+npm install
+bundle exec bowndler update
+bundle exec rake db:drop db:create db:schema:load db:migrate
+bundle exec rake
 bundle exec brakeman -q --no-pager --ensure-latest
 bundle exec rubocop .
-bundle exec rspec -f progress
-bundle exec cucumber -f progress
```

Comment:
Can we add rspec and cucumber back in as well please?

Reply:
`line 13` runs both spec and cucumber tests.